clang/unittests/Sema/HeuristicResolverTest.cpp

```diff
@@ -41,7 +41,7 @@ template <typename InputNode, typename ParamT, typename InputMatcher,
           typename... OutputMatchers>
 void expectResolution(llvm::StringRef Code, ResolveFnT<ParamT> ResolveFn,
                       const InputMatcher &IM, const OutputMatchers &...OMS) {
-  auto TU = tooling::buildASTFromCodeWithArgs(Code, {"-std=c++23"});
+  auto TU = tooling::buildASTFromCodeWithArgs(Code, {"-std=c++20"});
```

Comment:
This change (from c++20 to c++23) should be kept.

(It looks like the `this auto` test case manages to pass even without it, but that looks like a bug in the test suite; code with error diagnostics should cause the test to fail.)